Core/Config.cpp

```diff
@@ -423,7 +423,7 @@ static ConfigSetting generalSettings[] = {
 	ConfigSetting("FirstRun", &g_Config.bFirstRun, true),
 	ConfigSetting("RunCount", &g_Config.iRunCount, 0),
 	ConfigSetting("Enable Logging", &g_Config.bEnableLogging, true),
-	ConfigSetting("AutoRun", &g_Config.bAutoRun, true),
+	ConfigSetting("AutoRun", &g_Config.bAutoRun, false),
```

Comment:
We definitely don't want to change the default like this, or change its meaning.  People will get super confused if games don't start by default.

That's what this setting does: it's mainly for debugging.  If you're a rom hacker, you do want the game to wait for you to do stuff in the debugger before it starts running.  But that's not most people.

-[Unknown]